dto/src/main/java/org/jboss/pnc/dto/requests/BuildConfigWithSCMRequest.java

```diff
@@ -37,12 +38,21 @@
 @JsonDeserialize(builder = BuildConfigWithSCMRequest.Builder.class)
 public class BuildConfigWithSCMRequest {
 
+    /**
+     * The SCM repository url.
+     */
     @NotBlank
     @SCMUrl
     private final String scmUrl;
 
+    /**
+     * If the prebuild sync should be enabled if the SCM repository points to external url. Defaults to true.
```

Comment:
A javadoc shouldn't start with "if" IMO :-)
I would make it something like `Pre-builds sync enablement flag. Is taken into account only when scmUrl contains an external url. Defaults to true if null.`